locales/ko_KR/LC_MESSAGES/duckduckgo.po

```diff
@@ -3907,14 +3907,14 @@ msgstr "공연"
 #. Discontinued, don't translate
 msgctxt "settings"
 msgid "Shows the search button background"
-msgstr "뒤에 검색 버튼 보이기"
+msgstr ""
```

Comment:
It looks like you might have accidentally removed this translation without adding yours. Please, check. Thanks!

Reply:
I might have actually removed that by mistake. Thank you!